eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlResource.java

```diff
@@ -18,19 +17,23 @@
 @Produces(MediaType.APPLICATION_JSON)
 public class EidasSamlResource {
 
-    public EidasSamlResource() {
+    private EidasAuthnRequestValidator eidasAuthnRequestValidator;
+    private AuthnRequest authnRequest;
```

Comment:
Does `private AuthnRequest authnRequest` need to be a field on `EidasSamlResource`?

Reply:
Nope. It can be moved to the post method. I can add that in next PR?